src/Dimensions/dimension.jl

```diff
@@ -303,22 +303,25 @@ Base.size(dims::DimTuple) = map(length, dims)
 Base.CartesianIndices(dims::DimTuple) = CartesianIndices(map(d -> axes(d, 1), dims))
 
 # Extents.jl
+#=
 function Extents.extent(ds::DimTuple, args...)
     extent_dims = _astuple(dims(ds, args...))
     extent_bounds = bounds(extent_dims)
     return Extents.Extent{name(extent_dims)}(extent_bounds)
 end
+=#
 
-function _experimental_extent(ds::DimTuple)
-    regulardims = dims(ds, x -> !(lookup(x) isa MultiDimensionalLookup))    
-    regular_bounds = bounds.(regulardims)
+function Extents.extent(ids::DimTuple, args...)
+    ds = _astuple(dims(ids, args...))
+    regulardims = dims(ds, x -> !hasmultipledimensions(lookup(x)))    
+    regular_bounds = map(bounds, regulardims)
     regular_bounds_nt = NamedTuple{map(name, regulardims)}(regular_bounds)
     
     multidims = otherdims(ds, regulardims)
-    multidim_raw_bounds = bounds.(multidims) # we trust that bounds will give us a tuple of bounds one for each enclosed dimension
+    multidim_raw_bounds = map(bounds, multidims) # we trust that bounds will give us a tuple of bounds one for each enclosed dimension
```

Comment:
This needs to be made into a contract, perhaps in the docs of hasmultipledimensions and MultiDimensionalLookup